### PR TITLE
Added type definitions of string and Socket to database.ts

### PR DESCRIPTION
**Pull request by mssiegel**
**Test plan:** I chatted as several test users and the app worked as normal.

### CODE REVIEW

File: services/database.ts [hunk 1]
```diff
@@ -1,13 +1,15 @@
+import { Socket } from 'socket.io';
+
```
Comment (karlrez) [inline services/database.ts:2]:
```suggestion

interface Teacher {
  socket: Socket;
  classroomName: string;
}

```

Comment (karlrez) [reply services/database.ts:2]: Whats your thoughts on adding a type for teacher?

image

**Review by karlrez — APPROVED** [review]
I just left a single suggestion, all the other updates look good.

File: services/database.ts (hunk 1 above)
Comment (mssiegel) [reply services/database.ts:2]: We should! we should add types for classrooms, teachers, students, etc. all the objects listed at the top of `/database.ts`
I am just feeling rather lazy now hahahah. So thats why I only addressed strings and Socket in this PR.

**Pull request merged**